<ui version="4.0" >
 <class>menu</class>
 <widget class="QWidget" name="menu" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>menu</string>
  </property>
  <widget class="QLabel" name="label_1" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>5</y>
     <width>120</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Bitstream Charter</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>菜单</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>75</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="editable" >
    <bool>false</bool>
   </property>
   <item>
    <property name="text" >
     <string>Type all</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>45</y>
     <width>280</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text" >
    <string>欢迎光临，祝您用餐愉快！</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>120</y>
     <width>580</width>
     <height>230</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>365</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <underline>false</underline>
    </font>
   </property>
   <property name="text" >
    <string>编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>365</y>
     <width>65</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>495</x>
     <y>365</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <underline>false</underline>
    </font>
   </property>
   <property name="text" >
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3" >
   <property name="geometry" >
    <rect>
     <x>545</x>
     <y>365</y>
     <width>65</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="frame" >
    <bool>true</bool>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>410</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>订单预览</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>580</x>
     <y>75</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>540</x>
     <y>75</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <underline>false</underline>
    </font>
   </property>
   <property name="text" >
    <string>No</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>365</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>365</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>75</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
